sound/soc/intel/boards/sof_rt5682.c

```diff
@@ -24,10 +24,11 @@
 
 #define SOF_RT5682_MCLK_EN		BIT(0)
 #define SOF_RT5682_MCLK_24MHZ		BIT(1)
-#define SOF_RT5682_SSP0			BIT(2)
-#define SOF_RT5682_SSP1			BIT(3)
+#define SOF_RT5682_LEGACY_PLATFORM	BIT(2)
```

Comment:
can we also test the CPU ID instead of having a static quirk which is set by DMI checks? That way we can avoid having quirks for all legacy platforms, the legacy would be set dynamically.

Reply:
@plbossart Yes, sounds good